src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -2243,35 +2243,52 @@ public async Task When_LargeExtent_And_Very_Large_List_Scroll_To_End_And_Back_Ha
 
 			var scroll = list.FindFirstDescendant<ScrollViewer>();
 			Assert.IsNotNull(scroll);
-			dataContextChanged.Should().BeLessThan(10, $"dataContextChanged {dataContextChanged}");
```

Comment:
@Xiaoy312 should we keep the original test or is it broken with your changes?

Reply:
the original test were broken, hence why ive updated the test "metrics".
the primary behavior changes, that broke it, is that when scrolling with dynamic seed (AKA where to start adding items)
we no longer just add items forward, but also backward when necessary.
while during continuous scrolling/viewport changes, back filling is a waste, but it prevents the original issue from happening.

the old expectation were based on items within: [ScrollOffset, ScrollOffset + VP + EVP]
but in reality, it should be more within: [ScrollOffset **- EVP**, ScrollOffset + VP + EVP]